INSTALLATION.md

```diff
@@ -32,7 +32,7 @@ This document provides instructions on how to set up and start a running instanc
 
 Talawa-admin needs URL endpoint to a running instance of `talawa-api` graphql service to perform its operations. You need to set up a local instance of talawa-api on your system.
 
-Follow the [installation guide](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md) on talawa-api repo to set it up.
```

Comment:
Follow the instructions in the INSTALLATION.md file of the [Talawa-API repo](https://github.com/PalisadoesFoundation/talawa-api)"

Please update in all the places reported